.circleci/config.yml

```diff
@@ -269,6 +269,32 @@ jobs:
       - store_test_results:
           path: target/reports/
 
+  itest-events-v2-provider:
+    executor: ubuntu-machine-amd64
+    working_directory: /tmp/workspace/repo
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - prepare-pytest-tinybird
+      - prepare-account-region-randomization
+      - run:
+          name: Test EventBridge v2 provider
+          environment:
+            PROVIDER_OVERRIDE_EVENTS: "v2"
+            TEST_PATH: "tests/aws/services/events/"
```

Comment:
Do we have a single test that could pass, maybe from https://github.com/localstack/localstack/pull/10552?
I guess all would be skipped, for now, right?

Reply:
I will rebase once https://github.com/localstack/localstack/pull/10552 is merged, to include the single passing test `test_events.TestEventsEventBus.test_create_custom_event_bus`